src/accordion/accordion-section.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import Button from '../button/button.svelte';
+  import { createEventDispatcher } from 'svelte';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+
+  export let label = null;
+  export let open = false;
+
+  const selfControl = {
```

Comment:
I don't really see the need for an object here. It's not like we have a lot of them and need to organize internal states separately

Reply:
The need for an object is simply for guaranteed identification of each panel. Objects have referential equality, therefore, each panel is uniquely identified by its `selfControl` object, which is the behaviour that `<Accordion>` relies on